src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
You can add a class for html, but you don't need to create a BEM component from it. Usually, the components are made inside the body.